tap_gitlab/__init__.py

```diff
@@ -23,7 +23,9 @@
     'groups': '',
     'ultimate_license': False,
     'fetch_merge_request_commits': False,
-    'fetch_pipelines_extended': False
+    'fetch_pipelines_extended': False,
+    'fetch_group_variables': False,
+    'fetch_project_variables': False
```

Comment:
Ditto trailing comma

```suggestion
    'fetch_project_variables': False,
```